core/ViewDataTable/Config.php

```diff
@@ -363,6 +363,11 @@ class Config
     public $show_ecommerce = false;
 
     /**
+     * Stores an HTML message (if any) to display above the datatable view.
+     */
+    public $show_header_message = false;
```

Comment:
Can we add a little message to `show_header_message` and `show_footer_message` that content needs to be escaped properly as both print it raw?

Be also good to have this in developer changelog

Reply:
done :)